packages/popup/test/lion-popup.test.js

```diff
@@ -44,6 +45,21 @@ describe('lion-popup', () => {
       await el.updateComplete;
       expect(el.querySelector('strong')).to.not.be.undefined;
     });
+
+    it('should respond to dynamically changing the placementConfig', async () => {
+      const el = await fixture(html`
+        <lion-popup>
+          <div slot="content" class="popup">Hey there</div>
+          <lion-button slot="invoker">Popup button</lion-button>
+        </lion-popup>
+      `);
+      await el._controller.show();
+      expect(el._controller._popper.options.placement).to.equal('top');
+      el.placementConfig = { placement: 'left' };
+      // placementConfig setter calls updateConfig method which recreates popper instance and this process is async..
+      await aTimeout();
```

Comment:
isn't it better to hide/show and await for it instead of a hacky timeout?

Reply:
good point, actually, `await el._controller.show();` should be enough already by itself.